src/loaders/ObjectLoader.js

```diff
@@ -466,8 +466,21 @@ THREE.ObjectLoader.prototype = {
 
 				case 'PerspectiveCamera':
 
-					object = Object.assign(
-							new THREE.PerspectiveCamera(), data );
+					object = new THREE.PerspectiveCamera(
+							data.fov, data.aspect, data.near, data.far );
+
+					if ( data.focus !== undefined ) object.focus = data.focus;
+					if ( data.zoom !== undefined ) object.zoom = data.zoom;
+
+					if ( data.filmGauge !== undefined ) {
+
+						if ( data.view !== null )
+							object.view = Object.assign( {}, data.view );
```

Comment:
@bhouston 
>  This is so minor

As things are now, and considering the time already spent on this issue, I'd like to disagree and insert "seemingly" into that statement :)

There used to be an attempt of strict versioning in the past, but as we both observed, it isn't all too practical. 

So we're in fact talking about a policy that allows us to
- extend without making assumptions about the completeness of exporters, and
- making it easy enough, so that there's no temptation to sacrifice this goal in favor of lean code.

I think above `THREE.Utils.assignOptionals` (the name is probably bad), could safe the day when broadly applied in the loaders.